src/gwt/src/org/rstudio/studio/client/common/spelling/TypoSpellChecker.java

```diff
@@ -274,6 +275,15 @@ private boolean checkCustomDicts(String word)
 
    public void checkSpelling(List<String> words, final ServerRequestCallback<SpellCheckerResult> callback)
    {
+      // If realtime is turned off call the backend spellchecker because we haven't loaded a
+      // dictionary on the frontend. Ideally we DO load the frontend dictionary but currently
+      // Typo.js isn't compatible with some of our dictionaries (see: TypoSpellChecker.realtimeBlacklist)
+      // so we need to keep the legacy call for now.
+      if (!userPrefs_.realTimeSpellchecking().getValue()) {
```

Comment:
Nit: brace style

Reply:
You get too used to using your preferred brace style in JavaScript for a while and this happens...